Source/WebCore/platform/FileChooser.cpp

```diff
@@ -58,10 +58,6 @@ void FileChooser::chooseFile(const String& filename)
 
 void FileChooser::chooseFiles(const Vector<String>& filenames, const Vector<String>& replacementNames)
 {
-    // FIXME: This is inelegant. We should not be looking at settings here.
```

Comment:
the removal of this is necessary since the comparison of just file paths is no longer sufficient

Reply:
Do you ... know why it was here?